applications/FluidDynamicsApplication/python_scripts/stokes_solver_monolithic.py

```diff
@@ -1,228 +1,163 @@
 # importing the Kratos Library
-import KratosMultiphysics as kratoscore
-import KratosMultiphysics.python_linear_solver_factory as linear_solver_factory
-import KratosMultiphysics.FluidDynamicsApplication as cfd
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication
+
+# Import base class file
+from KratosMultiphysics.FluidDynamicsApplication.navier_stokes_solver_vmsmonolithic import StabilizedFormulation
+from KratosMultiphysics.FluidDynamicsApplication.navier_stokes_solver_vmsmonolithic import NavierStokesSolverMonolithic
+
+class StabilizedStokesFormulation(StabilizedFormulation):
```

Comment:
IMO this would be a good candidate to be refactored (future PR) into smaller files (and putting them in a subfolder of `python_scripts`)

Reply:
Good idea. I'd add it to the fluid project. Thanks!